packages/heartwood-components/components/feed/feed.scss

```diff
@@ -0,0 +1,69 @@
+// .message-feed__wrapper {
+// 	position: relative;
+// 	margin-top: rem(-48);
+// }
+
+// .message-feed {
+// 	display: flex;
+// 	flex-flow: column-reverse;
+// 	position: absolute;
+// 	top: 0;
+// 	left: 0;
+// 	width: 100%;
+// 	height: 100vh;
+// 	overflow-x: hidden;
+// 	overflow-y: scroll;
+
+// 	> *:last-child {
+// 		padding-top: rem(64);
+// 	}
+// }
+
+// .message-feed__virtual-list {
+// 	.ReactVirtualized__Grid__innerScrollContainer {
+// 		transform: rotate(180deg);
+// 	}
+// }
+
+// .message-feed {
+// 	transform: rotate(180deg);
+// }
+
+// .message-feed__message-wrapper {
+// 	transform: rotate(180deg);
+// }
+
+// .message-feed {
+// 	height: calc(100vh - #{rem(48)});
+// }
+
+// .message-feed,
+// .message-feed__message-wrapper {
+// 	transform: scaleY(-1);
+// }
```

Comment:
@jtini can the commented code be removed?

Reply:
yep - will remove